### PR TITLE
[PfemFluid] Cleaning of strategy and bugfix 

### PR DESCRIPTION
**Pull request by AFranci**
Addressing Rafael's and Alejandro's comments.

### CODE REVIEW

**Comment by AlejandroCornejo** [comment]
The tests are affected? Just curiosity :D

**Review by rlrangel — APPROVED** [review]
Working better. Now the iterative loop breaks when the tolerance is reached.

**Comment by AlejandroCornejo** [comment]
> 
> 
> Working better. Now the iterative loop breaks when the tolerance is reached.

Happy to read that!!

**Review by AlejandroCornejo — APPROVED** [review]
Good

**Comment by AFranci** [comment]
> The tests are affected? Just curiosity :D

yes!

File: applications/PfemFluidDynamicsApplication/custom_strategies/strategies/two_step_v_p_strategy.h [hunk 1]
```diff
@@ -211,7 +211,6 @@ namespace Kratos
 
     double Solve() override
```
Comment (rubenzorrilla) [inline applications/PfemFluidDynamicsApplication/custom_strategies/strategies/two_step_v_p_strategy.h:212]: Just for you to know as I see that you are trying to clean up the strategy. The `Solve` method shouldn't be no longer used. Instead the `InitializeSolutionStep`, `SolveSolutionStep` ... should be employed.

Comment (AFranci) [reply applications/PfemFluidDynamicsApplication/custom_strategies/strategies/two_step_v_p_strategy.h:212]: Thank you Ruben, we will update it too, also in the other strategies.
In the meantime, I merge this pull request as it solves a bug..

**Pull request merged**